ledger-api/grpc-definitions/com/daml/ledger/api/v1/active_contracts_service.proto

```diff
@@ -29,10 +34,12 @@ message GetActiveContractsRequest {
 
   // Must correspond to the ledger ID reported by the Ledger Identification Service.
   // Must be a valid LedgerString (as described in ``value.proto``).
+  // ``NOT_FOUND`` will be returned in case the ledger does not exist.
```

Comment:
```suggestion
  // Returns ``NOT_FOUND`` if the ledger does not exist.
```

I personally prefer the shorter active format `"Returns <error code> if <condition>"`. This style is also used in [other places](https://github.com/digital-asset/daml/blob/master/ledger-api/grpc-definitions/com/daml/ledger/api/v1/command_service.proto#L23) in our protobuf comments.

Reply:
I like it more than the style I used for the PR. I'll change it.